Automattic/wp-calypso
# Fixes the relative date shown in Promote post

Related to SELFSERVE-628

## Proposed Changes

This PR fixes the relative time shown on the Advertising page. It uses a similar function to the one used on the posts page.
This way, it will feel normal to the users.

**NOTE:** This PR needs to be merged after this diff: D114400. If not, it will show incorrect relative times. 
We already are showing an incorrect time, but this change it make it more noticeable,

![Screenshot 2023-08-10 at 14 25 27](https://github.com/Automattic/wp-calypso/assets/1258162/a42a4c20-eb39-4201-8da5-3c4ca94be491)

![Screenshot 2023-08-10 at 14 19 31](https://github.com/Automattic/wp-calypso/assets/1258162/606d07f0-9d1d-415d-a157-3b5c884bd533)

## Testing Instructions

* Open the live preview and navigate to Tools->Advertising
* Verify that the correct relative time is shown when the difference is less than a week

## Pre-merge Checklist
